catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -275,6 +275,19 @@ public AttributeFormat getAttributeFormat() {
                 false /* tokenized */,
                 true /* multivalued */,
                 STRING_TYPE));
+        descriptors.add(new AttributeDescriptorImpl(Metacard.RESOURCE_CHECKSUM_ALGORITHM,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                STRING_TYPE));
+        descriptors.add(new AttributeDescriptorImpl(Metacard.RESOURCE_CHECKSUM,
```

Comment:
I don't have a strong opinion.  We decided on this attribute name when the checksum provider was initially created.  We should probably defer to the outcome of the taxonomy work Brendan is doing.